app.js

```diff
@@ -1,19 +1,37 @@
-const express = require('express')
-const app = express()
-app.use(express.json())
-const {getAllTopics, getEndpoints} = require('./controllers/topics.controllers')
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
Are you using this? (Hint - no you are not using this!) Please delete and only add code that is required. This is not good practice.